editor/plugins/tiles/tile_set_atlas_source_editor.h

```diff
@@ -288,7 +290,7 @@ class TileSetAtlasSourceEditor : public HSplitContainer {
 
 public:
 	void edit(Ref<TileSet> p_tile_set, TileSetAtlasSource *p_tile_set_source, int p_source_id);
-	void init_source();
+	void init_source(Vector<Ref<TileSetAtlasSource>> atlases_to_auto_create_tiles);
```

Comment:
```suggestion
	void init_source(const Vector<Ref<TileSetAtlasSource>> &p_atlases);
```
Match argument names